project.clj

```diff
@@ -5,6 +5,7 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.7.0"]
                  [org.clojure/clojurescript "1.7.170" :scope "provided"]
+                 [org.clojure/math.combinatorics "0.1.3" :scope "test"]
```

Comment:
To add "test scope" dependencies in leiningen you need to put them in a different profile.  The `:dev` profile is a common default one.  It'll be on the classpath when tests are ran automatically too.